dart/lib/src/enricher/web_enricher_event_processor.dart

```diff
@@ -1,17 +1,11 @@
 import 'dart:async';
-import 'dart:html' as html show window, Window;
+import 'dart:html' as html;
 
 import '../event_processor.dart';
 import '../protocol.dart';
 import '../sentry_options.dart';
 
-EventProcessor enricherEventProcessor(SentryOptions options) {
-  return WebEnricherEventProcessor(
-    html.window,
-    options,
-  );
-}
-
+// ignore: unused-code
```

Comment:
This is not needed if we solve https://github.com/getsentry/sentry-dart/pull/1081/files#r1000232135
Either we remove this integration or it should be added in the integration list